src/calculateRentalCost.js

```diff
@@ -4,7 +4,30 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = 0;
+  const costCar = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+
+  if (days < 3) {
+    sum = costCar * days;
+
+    return sum;
+  }
+
+  if (days >= 3 && days <= 6) {
+    sum = costCar * days - threeDaysDiscount;
+
+    return sum;
+  }
```

Comment:
The condition `days >= 3 && days <= 6` can be simplified to `days < 7` since you're already returning early if `days < 3`. This simplifies the logic and makes it clear that this block only runs for days between 3 and 6.